modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,300 @@
+
+
+@pytest.mark.parametrize(
+    "type_modin_y",
+    [pd.DataFrame, pd.Series],
+)
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, 400],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_iris(),
+            {
+                "max_depth": 3,
+                "eta": 0.3,
+                "objective": "multi:softprob",
+                "num_class": 3,
+            },
+        ),
+        (load_digits(), {"max_depth": 6, "num_class": 10, "n_estimators": 200}),
+        (
+            load_wine(),
+            {
+                "objective": "multi:softprob",
+                "eta": 0.4,
+                "max_depth": 3,
+                "gamma": 1.0,
+                "num_class": 3,
+            },
+        ),
+    ],
+    ids=[
+        "load_iris",
+        "load_digits",
+        "load_wine",
+    ],
+)
+def test_dmatrix_train_predict_multiclass_classification_datasets(
+    data, num_actors, num_round, type_modin_y
+):
+    dataset, param = data
+
+    X = dataset.data
+    y = dataset.target
+    dmatrix_xgb = xgboost.DMatrix(X, label=y)
+    with pytest.raises(AssertionError):
+        dmatrix_mxgb = xgb.DMatrix(X, label=y)
+
+    modin_X = pd.DataFrame(X)
+    modin_y = type_modin_y(y)
+    dmatrix_mxgb = xgb.DMatrix(modin_X, label=modin_y)
+
+    evals_result_xgb = {}
+    evals_result_mxgb = {}
+    bst = xgboost.train(
+        param,
+        dmatrix_xgb,
+        num_round,
+        evals_result=evals_result_xgb,
+        evals=[(dmatrix_xgb, "train")],
+    )
+    modin_bst = xgb.train(
+        param,
+        dmatrix_mxgb,
+        num_round,
+        evals_result=evals_result_mxgb,
+        evals=[(dmatrix_mxgb, "train")],
+        num_actors=num_actors,
+    )
+
+    predictions = bst.predict(dmatrix_xgb)
+    modin_predictions = modin_bst.predict(dmatrix_mxgb, num_actors=num_actors)
+
+    best_preds = np.asarray([np.argmax(line) for line in predictions])
+    modin_best_preds_ = np.asarray(
+        [np.argmax(line) for line in modin_predictions.to_numpy()]
+    )
+    modin_best_preds = pd.DataFrame(modin_best_preds_)
+
+    preds = pd.DataFrame(best_preds).apply(lambda x: round(x))
+    modin_preds = modin_best_preds.apply(lambda x: round(x))
+
+    val = accuracy_score(y, preds)
+    modin_val = accuracy_score(modin_y, modin_preds)
+
+    np.testing.assert_allclose(val, modin_val)
+
+
+@pytest.mark.parametrize(
+    "type_modin_y",
+    [pd.DataFrame, pd.Series],
+)
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, 400],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_diabetes(),
+            {
+                "n_estimators": 500,
+                "max_depth": 4,
+                "min_samples_split": 5,
+                "learning_rate": 0.01,
```

Comment:
Let's use xgboost-style of naming instead of sklearn:
```suggestion
                "eta": 0.01,
```

Reply:
This parameter was changed